tests/phpunit/class-test-core-sitemaps.php

```diff
@@ -159,5 +216,237 @@ public function test_core_sitemaps_xml() {
 		$this->assertSame( $expected, $xml, 'Sitemap page markup incorrect.' );
 	}
 
+	/**
+	 * Helper function to get all sitemap entries data.
+	 *
+	 * @return array A list of sitemap entires.
+	 */
+	public function _get_sitemap_entries() {
+		$entries   = array();
+
+		$providers = core_sitemaps_get_sitemaps();
+
+		foreach ( $providers as $provider ) {
+			$entries = array_merge( $entries, $provider->get_sitemap_entries() );
```

Comment:
For performance reasons I'd usually avoid `array_merge` in a loop. Here it should be OK though as it's not a lot of data.